maat-court-data-api/src/main/java/gov/uk/courtdata/billing/request/ApplicantResetRequest.java

```diff
@@ -0,0 +1,17 @@
+package gov.uk.courtdata.billing.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ApplicantResetRequest {
+
+    @NotBlank(message = "Username must not be blank")
+    private String username;
+
+    @NotEmpty(message = "Applicant IDs must not be empty")
+    private List<Integer> applicantIds;
+}
```

Comment:
I would make use of the existing `UpdateRepOrderBillingRequest` class for now and remove this one, as in Nigel's [pending PR](https://github.com/ministryofjustice/laa-maat-court-data-api/pull/1126) this class has been refactored to make a generic `UpdateBillingRequest` that can be used across different resources).